graphql/graphql.go

```diff
@@ -598,6 +619,19 @@ func (b *Block) BaseFeePerGas(ctx context.Context) (*hexutil.Big, error) {
 	return (*hexutil.Big)(header.BaseFee), nil
 }
 
+func (b *Block) NextBaseFeePerGas(ctx context.Context) (*hexutil.Big, error) {
+	header, err := b.resolveHeader(ctx)
+	if err != nil {
+		return nil, err
+	}
+	if header.BaseFee == nil {
```

Comment:
What if the current block is the last block before london fork. The next block is supposed to return `params.InitialBaseFee`?

Reply:
Perhaps something like this.

```go
var nextBaseFee *big.Int
if chain.Config().IsLondon(new(big.Int).Add(header.Number, common.Big1)) {
	nextBaseFee = misc.CalcBaseFee(chain.Config(), header)
}
```

Reply:
Good catch, fixed